### ISSUE TITLE
[error] State file error: duplicate app version: setiathome_enhanced windows_intelx86 513

### ISSUE THREAD

**Issue by romw** [BOINC/boinc-ng#267]
**Reported by mozeskriebels on 22 Dec 37452609 08:53 UTC**
The following error occurs when using the boinc version 5.10.2 and 5.10.6 with the optimized client (KWSN_2.2B_SSE2-P4_Ben-Joe.exe).
```
14-6-2007 15:29:49|SETI@home|[State file error: duplicate app version: setiathome_enhanced windows_intelx86 513
14-6-2007 15:29:49|SETI@home|[error](error]) State file error: duplicate app version: setiathome_enhanced windows_intelx86 514
14-6-2007 15:29:49|SETI@home|[State file error: duplicate app version: setiathome_enhanced windows_intelx86 515
14-6-2007 15:29:49|SETI@home|[error](error]) State file error: duplicate app version: 
setiathome_enhanced windows_intelx86 517
```
The APP_INFO.XML is beneeth:
```
<app_info>
    <app>
        <name>setiathome_enhanced</name>
    </app>
    <file_info>
        <name>KWSN_2.2B_SSE2-P4_Ben-Joe.exe</name>
        <executable/>
    </file_info>
    <app_version>
        <app_name>setiathome_enhanced</app_name>
        <version_num>512</version_num>
        <file_ref>
            <file_name>KWSN_2.2B_SSE2-P4_Ben-Joe.exe</file_name>
            <main_program/>
        </file_ref>
    </app_version>
    <app_version>
        <app_name>setiathome_enhanced</app_name>
        <version_num>513</version_num>
        <file_ref>
            <file_name>KWSN_2.2B_SSE2-P4_Ben-Joe.exe</file_name>
            <main_program/>
        </file_ref>
    </app_version>
    <app_version>
        <app_name>setiathome_enhanced</app_name>
        <version_num>514</version_num>
        <file_ref>
            <file_name>KWSN_2.2B_SSE2-P4_Ben-Joe.exe</file_name>
            <main_program/>
        </file_ref>
    </app_version>
    <app_version>
        <app_name>setiathome_enhanced</app_name>
        <version_num>515</version_num>
        <file_ref>
            <file_name>KWSN_2.2B_SSE2-P4_Ben-Joe.exe</file_name>
            <main_program/>
        </file_ref>
    </app_version>
    <app_version>
        <app_name>setiathome_enhanced</app_name>
        <version_num>517</version_num>
        <file_ref>
            <file_name>KWSN_2.2B_SSE2-P4_Ben-Joe.exe</file_name>
            <main_program/>
        </file_ref>
    </app_version>
</app_info>
```

**Comment by romw** [comment]
**Commented by Nicolas on 24 Aug 37453080 02:13 UTC**
8.10.6 doesn't exist, I guess you meant 5.10.6. I'll fix the description.

**Comment by romw** [comment]
**Commented by davea on 26 Oct 37453442 04:26 UTC**
Fixed in june 13 07 checkin (after 5.10.6)
